tools/http_benchmark.py

```diff
@@ -138,6 +148,7 @@ def http_benchmark(build_dir):
     return {
         # "deno_tcp" was once called "deno"
         "deno_tcp": deno_tcp(deno_exe),
+        # "deno_udp": deno_udp(deno_exe),
```

Comment:
Why disabled?

Reply:
Because the benchmark script works with a HTTP server, which relies on TCP, so it does not work with UDP, therefore I disabled it until the benchmark script supports UDP.

Reply:
In that case can you remove this and tools/deno_udp ?

Reply:
Yes, I do it now